concent_api/core/validation.py

```diff
@@ -468,28 +467,17 @@ def substitute_new_report_computed_task_if_needed(
     return new_report_computed_task
 
 
-def validate_list_of_transaction_timestamp(
-    list_of_transactions: List[BatchTransferEvent],
-    acceptances: List[SubtaskResultsAccepted],
-) -> None:
+def validate_list_of_transaction_timestamp(acceptances: List[SubtaskResultsAccepted]) -> None:
     """
     Validate if cut_off_time < youngest_payment_ts(T2) + PDT, unless requestor already made some payment.
     We compare only youngest payment_ts, because we want to use the highest value of timestamp. Other payment_ts might
     give false positives.
     """
 
     cut_off_time = get_current_utc_timestamp()
-
     youngest_payment_ts = max(subtask_results_accepted.payment_ts for subtask_results_accepted in acceptances)
 
-    if (
-        (
-            any(youngest_payment_ts > closure_time for closure_time in list_of_transactions) or
-            len(list_of_transactions) == 0
-        ) and (
-            cut_off_time < youngest_payment_ts + settings.PAYMENT_DUE_TIME
-        )
-    ):
+    if cut_off_time < youngest_payment_ts + settings.PAYMENT_DUE_TIME:
```

Comment:
This condition would be easier to understand for me as:

``` python
if youngest_payment_ts > cut_off_time - settings.PAYMENT_DUE_TIME:
```
because the focus here is on `youngest_payment_ts`, not `cut_off_time` (which is just the current time).

Reply:
:+1: done